### DISCUSSION TITLE
/var/log  Files "truncate" i.e. empty contents, after one hour

### DISCUSSION THREAD

**Discussion by jalsco** [MichaIng/DietPi#6232] [Q&A]
I am on latest ver of DietPi , and everything running very smoothly  ( with eth0 and eth1 ). 

Hoping this is a very easy question.  Am running a 1 minute cron job to log something, for my own diagnoistic purposes related to my network traffic... Anyway I  was writing the log file to `/var/log/<mylog-name>.log`  --- and I notice that it "truncates" , that is wipes out its contents every hour exactly  ( I think about 2 minutes, or 2 seconds,  after the hour ) .... 

I did my best to investigate the usual suspects : 
-  hourly cron job
-  some `logrotate` setting or defaults or conf file.. 
-  Anything :  `lsof <path_to_log_file>` else access  the file ?
-  even trying to see what going on `watch -n 1 "ps aux | grep 'eth1-2023-03-09.log'"`

But I just could not get to the bottom of it.   In desperation I just moved my log file to my own directory ( actually moved to `/root/log`) and the log files never get truncated while in that place.  

Obviously something is set to a one hour timer to "manage"  log files in `/var/log` - what is it and how to adjust those settings ?

**Comment by Joulinar** [comment]
This is working as expected. You would need to change your Log System Choice https://dietpi.com/docs/software/log_system/#log-system-choices
